maintainers/maintainer-list.nix

```diff
@@ -9330,6 +9330,20 @@
     github = "istoph";
     githubId = 114227790;
   };
+  itepastra = {
+    name = "Noa Aarts";
+    github = "itepastra";
+    githubId = 27058689;
+    email = "itepastra@gmail.com";
+    keys = [
+      {
+        fingerprint = "E681 4CAF 06AE B076 D55D  3E32 A16C DCBF 1472 541F";
+      }
+      {
+        fingerprint = "F960 8602 323A F7AB 7603  43C9 A9DD F987 11FA B741";
+      }
```

Comment:
can you drop this second fingerprint? The first fingerprint is the one used to sign the two commits in this PR so that's the only one I can readily verify matches.